Specs/DateTime/Chinese/DateTimeModel.json

```diff
@@ -5875,6 +5882,7 @@
     "Context": {
       "ReferenceDateTime": "2017-03-22T00:00:00"
     },
+    "NotSupported": "javascript",
```

Comment:
How much effort would be required to
1- make such isLunar tests pass in JS?
2- implement the same mods in this PR in JS?

Reply:
not much in theory:
for the "isLunar" tag, one additional "addResolutionFieldsAny" line should do it;
for the mods behaviour, we replaced some of the Chinese parsing method with the corresponding standard methods, the same should be doable in JS.

Reply:
should we proceed with this?

Reply:
Please do if/whenever no pending P1/P2 issues. Thanks.